test/cmdlineTests/storage_layout_user_defined_no_pretty_print/input.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL v3
+type MyInt128 is int128;
+type MyInt8 is int8;
+contract C {
+    // slot 0
+    MyInt128 a;
+    MyInt128 b;
+    // slot 1
+    MyInt128 c;
+    MyInt8 d;
+    MyInt8 e;
+    MyInt8 f;
+    MyInt8 g;
+    // slot 2
+    MyInt8 h;
+}
```

Comment:
This test is just for JSON printing so it can be very simple:
```suggestion
contract C {}
```
We only really care that it prints the output from `--storage-layout` and that that output is not prettified so no point having much more here.